wrapper/src/main/java/software/amazon/jdbc/DataSourceConnectionProvider.java

```diff
@@ -191,16 +168,7 @@ public Connection connect(
   @Deprecated
   public Connection connect(final @NonNull String url, final @NonNull Properties props) throws SQLException {
     final Properties copy = PropertyUtils.copyProperties(props);
-
-    if (!isNullOrEmpty(this.urlPropertyName)) {
-      copy.setProperty(this.urlPropertyName, url);
-    }
-
-    if (!isNullOrEmpty(this.databasePropertyName)
-        && !isNullOrEmpty(PropertyDefinition.DATABASE.getString(props))) {
-      copy.put(this.databasePropertyName, PropertyDefinition.DATABASE.getString(props));
-    }
-
+    copy.setProperty("url", url);
```

Comment:
where is this property used/retrieved?

Reply:
This method is not in use and it's deprecated. We just assume that target data source has `url` property.